MAUI/ImageEditor/text.md

```diff
@@ -48,6 +48,7 @@ Customize the appearance of text annotation using the [`ImageEditorTextSettings`
 * [`IsEditable`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.ImageEditorTextSettings.html#Syncfusion_Maui_ImageEditor_ImageEditorTextSettings_IsEditable): Enables or disables text editing. The default value is `true`.
 * [`IsRotatable`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.ImageEditorTextSettings.html#Syncfusion_Maui_ImageEditor_ImageEditorTextSettings_IsRotatable): Enables or disables the text rotation. The default value is `true`.
 * [`TextAlignment`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.ImageEditorTextSettings.html#Syncfusion_Maui_ImageEditor_ImageEditorTextSettings_TextAlignment): Specifies the start, center, or end alignment of the text. The default value is `TextAlignment.Center`.
+* `Background`: Specifies the background color of the text annotation.
```

Comment:
<img width="547" alt="image" src="https://github.com/syncfusion-content/maui-docs/assets/92788387/eb81e482-4681-4519-971d-871af94b8b82">

Specify opacity is applied to background color.

Reply:
Specified a opacity for background color